packages/transfer-app/src/SendBalance.tsx

```diff
@@ -88,51 +88,51 @@ export class SendBalance extends React.PureComponent<Props, State> {
     const { amount, isAddressValid, recipientAddress } = this.state;
 
     return (
-        <Grid>
-          <Grid.Row centered>
-            <Header> Transfer Balance </Header>
-          </Grid.Row>
-          <Grid.Row>
-            <Grid.Column width='6'>
-              <Stacked>
-                <Step.Group vertical>
-                  <Step completed={isAddressValid}>
-                    <Step.Title> Recipient </Step.Title>
-                    <Icon name='address book' color='blue' />
-                    <Step.Content>
-                      <MarginTop />
-                      <Stacked>
-                        <AddressSummary address={recipientAddress} size='small' />
-                        <Input onChange={this.onChangeRecipientAddress} type='text' value={recipientAddress} />
-                      </Stacked>
-                    </Step.Content>
-                  </Step>
-
-                  <Step completed={!amount.isZero()}>
-                    <Step.Title> Amount </Step.Title>
-                    <Icon name='law' color='blue' />
-                    <Step.Content>
-                      <Stacked>
-                        <Input onChange={this.onChangeAmount} type='number' value={amount} />
-                      </Stacked>
-                    </Step.Content>
-                  </Step>
-
-                  <Step>
-                    <Icon name='send' color='blue' />
-                    <Step.Content>
-                      <NavButton onClick={this.onSubmitTransfer}>Submit Transfer</NavButton>
-                    </Step.Content>
-                  </Step>
-                </Step.Group>
-              </Stacked>
-            </Grid.Column>
-
-            <Grid.Column width='10'>
-              <Saved onSelectAddress={this.onSelectAddress} {...this.props} />
-            </Grid.Column>
-          </Grid.Row>
-        </Grid>
+      <Grid>
```

Comment:
weird that our ide linters have different behaviors, and that tslint doesn't shout (it should be 2 spaces)

Reply:
That's my bad I think. Just looked into it and I'm using `language-typescript v0.4.11` for Atom. Went ahead and uninstalled that in favor of `atom-typescript v13.1.0`